data/templates/get_driveid.js

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env node
+
+'use strict';
+
+var exec = require('child_process').exec;
+
+var cmdDriveWwid = 'ls -l /dev/disk/by-id';
+var cmdVdInfo = 'ls -l /dev/disk/by-path';
+var cmdScsiId = 'lsscsi';
+var options = {
+    timeout: 10000
+};
+
+/**
+ * Parse the Drive WWID output
+ * @param {String} data
+ * @return
+ */
+function parseDriveWwid(idList) {
+    //idList example
+    //lrwxrwxrwx. 1 root root  9 Nov 18 20:30 ata-SATADOM-SV_3SE_20150522AA9992050085 -> ../../sdb
+    var lines = idList.split('\n').map(function(line) {
+        var split = line.split(/\s+/);
+        return [split[8],split[10]].join('->');
```

Comment:
Could you please add a comment to show an example for the output? It's hard for uses to know the what split[8] and split[10] means.